src/components/ItemSelector/ItemSelector.js

```diff
@@ -25,7 +24,7 @@ const ItemSelector = ({}, context) => {
 
         const filterStr = filter ? `/${filter}` : ''
 
-        context.d2.Api.getApi()
+        d2.Api.getApi()
```

Comment:
@edoardo Could dataEngine be used here instead?

Reply:
I think it should be possible, although with a hacky way of dealing with that URL.